tests/back/Pim/Structure/Integration/Attribute/Query/GetAllBlacklistedAttributeCodesIntegration.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+declare(strict_types=1);
+
+namespace AkeneoTest\Pim\Structure\Integration\Family;
+
+use Akeneo\Pim\Structure\Bundle\Manager\AttributeCodeBlacklister;
+use Akeneo\Pim\Structure\Bundle\Query\InternalApi\Attribute\GetAllBlacklistedAttributeCodes;
+use Akeneo\Test\Integration\Configuration;
+use Akeneo\Test\Integration\TestCase;
+use Doctrine\DBAL\Connection;
+
+final class GetAllBlacklistedAttributeCodesIntegration extends TestCase
+{
+    private Connection $sqlConnection;
+
+    public function setUp(): void
+    {
+        parent::setUp();
+
+        $this->sqlConnection = $this->get('database_connection');
+    }
+
+    public function test_it_returns_an_array_of_blacklisted_attributes(): void
+    {
+        $blacklister = $this->getBlacklister();
+        $blacklister->blacklist('description');
+        $blacklister->blacklist('name');
+
+        $query = $this->getQuery();
+        $result = $query->execute();
+
+        $this->assertEquals($result, ['description', 'name']);
```

Comment:
```suggestion
        $this->assertEquals(['description', 'name'], $result);
```
Not important at all but it should be expected first

Reply:
ok, perfect, I will change that 👍